packages/terra-outline-view-table/src/terra-dev-site/doc/outline-view-table/example/DefaultTable.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import OutlineTableView from 'terra-outline-table-view';
+import OutlineViewTable from 'terra-outline-table-view';
```

Comment:
Should this be changed as well?
```suggestion
import OutlineViewTable from 'terra-outline-view-table';
```

Reply:
Updated https://github.com/cerner/terra-core/pull/3692/commits/ba8ab299b27ddc4eb14861ac8624ed6e197bcb16